local.data.json

```
@@ -4,6 +4,11 @@
       "id": "article380046",
       "url": "https://travail-emploi.gouv.fr/droit-du-travail/les-contrats-de-travail/article/cdd-multi-remplacement-relance-de-l-experimentation-questions-reponses",
       "updated_at": "2023-03-14T10:09:43Z"
+    },
+    {
+      "id": "article-1",
```

```suggestion
      "id": "article-ACFg1E",
```

Je génère un id unique, car on sait jamais si un jour on aura une collision